### ISSUE TITLE
App build crashes after 3.2 update (Android)

### ISSUE THREAD

**Issue by bachras** [NativeScript/nativescript-background-http#118]
Hi,

Plugin was working perfectly fine until update (3.2 and 3.2.1) doesn't work anymore. Here is the error I get when I try to run 'tns run android' (after removing platform, node_modules, hooks):
````
/Users/john/test/platforms/android/app/build/intermediates/res/merged/F0F1F2F3F4F5F6/debug/values-v26/values-v26.xml:15:21-54: AAPT: No resource found that matches the given name: attr 'android:keyboardNavigationCluster'.
    
/Users/john/test/platforms/android/app/build/intermediates/res/merged/F0F1F2F3F4F5F6/debug/values-v26/values-v26.xml:15: error: Error: No resource found that matches the given name: attr 'android:keyboardNavigationCluster'.



FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processF0F1F2F3F4F5F6DebugResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt
````

Did anybody else had same issue?
P.s. nativescript and android platform is up-to-date.

Any help/advice is highly appreciated. Thank you

**Comment by radeva** [comment]
Hi @bachras ,

Could you delete your `platforms` folder and try again? It's a know issue that when a native dependency is updated, you need to remove your `platforms` and run the app again.

**Comment by bachras** [comment]
Hi @radeva,

I have mentioned in my previous post, that I tried to remove platforms, node_modules and hooks folders and package-lock.json file and run ```tns run android``` several times. I also researched for error online everywhere, so I tried removing *.gz, *.tar, *.zip as @NathanaelA  suggested in https://github.com/NativeScript/NativeScript/issues/2686 . Tried clearing android and npm caches. Nothing helped, only changing to 3.1.x version helped. I am really happy with the plugin, therefore I would like to get it fixed and install the latest version. Thank you for taking time to look at the issue.
